src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -371,14 +369,92 @@ private extension UITableViewCell {
 
 		switch location {
 		case .top:
-			separator.tag = SeparatorLocation.top.rawValue
+			separator.tag = CellSeparatorLineLocation.top.rawValue
 			separator.topAnchor.constraint(equalTo: contentView.topAnchor).isActive = true
 		case .bottom:
-			separator.tag = SeparatorLocation.bottom.rawValue
+			separator.tag = CellSeparatorLineLocation.bottom.rawValue
 			separator.bottomAnchor.constraint(equalTo: contentView.bottomAnchor).isActive = true
 		case .inBetween:
-			separator.tag = SeparatorLocation.inBetween.rawValue
+			separator.tag = CellSeparatorLineLocation.inBetween.rawValue
 			separator.bottomAnchor.constraint(equalTo: contentView.bottomAnchor).isActive = true
 		}
 	}
 }
+
+/// Stuff to draw a background to the cell, depending on the position of the cell in the section it gets rounded corners.
+
+private extension UITableViewCell {
+	
+	func drawBackground(
+		section: DynamicSection,
+		at indexPath: IndexPath
+	) {
+		self.removeBackground()
+		
+		switch section.background {
+		case .none:
+			break
+		case .greyBoxed:
+			
+			// Give all subviews in the content view of the cell some insets to the border. If we change the contentView's constraints, it would not affect the subviews.
```

Comment:
```suggestion
			// Give the root view in the content view of the cell some insets to the border. If we change the contentView's constraints, it would not affect the subviews.
```